examples/chart/teleport-cluster/templates/deployment.yaml

```diff
@@ -190,9 +190,9 @@ spec:
         configMap:
           name: {{ .Release.Name }}
       - name: "data"
-        {{- if eq .Values.chartMode "standalone" }}
+        {{- if or (eq .Values.chartMode "standalone") (eq .Values.chartMode "custom") }}
         persistentVolumeClaim:
-          claimName: {{ if .Values.standalone.existingClaimName }}{{ .Values.standalone.existingClaimName }}{{ else }}{{ .Release.Name }}{{ end }}
+          claimName: {{ if .Values.persistence.existingClaimName }}{{ .Values.persistence.existingClaimName }}{{ else }}{{ .Release.Name }}{{ end }}
```

Comment:
Another thing we'd also need to do is support backward compatibility for this, i.e. someone who still has `standalone.existingClaimName` set in their values shouldn't have their chart broken by this change. I'd suggest using `coalesce` to pick the first non-empty value in a similar way to https://github.com/gravitational/teleport/blob/master/examples/chart/teleport-cluster/templates/config.yaml#L2